galaxyproject/galaxy
# Add tooltip preview for rules in SavedRulesSelector

This PR is a solution for #13428 
It adds a tooltip that briefly highlights 5 of the rules/mappings that are present in a tool.  

![Screenshot from 2022-02-24 10-10-29](https://user-images.githubusercontent.com/26912553/155551892-1e2e9fe1-7137-49b1-991d-2918a9e2ab68.png)



## How to test the changes?
(Select all options that apply)
  1. create some rules/mappings using the rule builder.
  2. Hover over them in the saved rule drop down to see a brief description of what you did. 

## License
